src/components/ContextMenuPopover/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {createPortal} from 'react-dom';
+import {propTypes, defaultProps} from '../Popover/PopoverPropTypes';
+import withWindowDimensions from '../withWindowDimensions';
+import Popover from '../Popover';
+import CONST from '../../CONST';
+
+const ContextMenuPopover = props => (!props.isSmallScreenWidth ? createPortal(
```

Comment:
Why creating a portal? Some `zIndex` business? Can we leave a comment? Should we just make this change directly in `Popover` instead of creating a new component?

Reply:
Z-index does not work here. z-index is always relative to the parent. If its parent sibling has a higher z-index that will cover the Menu.